admin/python/lsst/qserv/admin/qservAdmin.py

```diff
@@ -314,7 +314,7 @@ def _createDbLikeLocked(self, dbName, templateDbName):
             self._createDbLockSection(dbP)
             self._kvI.set(dbP, "READY")
         except KvException as e:
-            self._logger.error("Failed to create database '%s' like '%s', error was: %s",
+            self._logger.error("Failed to create db '%s' like '%s', error was: %s",
```

Comment:
All right, I am replacing this consistently in all logging messages. If you think that is too much, let me know :)